api/k8s/v1/k8s.proto

```diff
@@ -419,6 +422,31 @@ message Deployment {
 
   map<string, string> labels = 4;
   map<string, string> annotations = 5;
+
+  message DeploymentStatus {
+    uint32 replicas = 1;
+    uint32 updated_replicas = 2;
+    uint32 ready_replicas = 3;
+    uint32 available_replicas = 4;
+    uint32 unavailable_replicas = 5;
+    message DeploymentCondition {
```

Comment:
Having Deployment here feels a little redundant since this is in the `DeploymentStatus` message. Maybe replace this with just `Condition` and same comment for the `Condition Type`